dockerfiles/ubuntu/obbi/worker/Dockerfile

```diff
@@ -17,9 +17,9 @@
 # ------------------------------------------------------------------------
 
 # set base Docker image to AdoptOpenJDK Ubuntu Docker image
-FROM adoptopenjdk:8u242-b08-jdk-hotspot-bionic
+FROM adoptopenjdk:8u262-b10-jdk-hotspot-bionic
```

Comment:
@nalakagunathilaka the update number of AdoptOpenJDK has to be uniform -> please use the [focal image](https://hub.docker.com/layers/adoptopenjdk/library/adoptopenjdk/8u272-b10-jdk-hotspot-focal/images/sha256-ec12ef973c20bda37ef09fb63477e62fe3d0784a3072609c91162c3e0d46ec39?context=explore).

Reply:
Resolved with latest commit.